directory/assets/service_checks.json

```diff
@@ -1 +1,11 @@
-[]
+[
+  {
+    "agent_version": "6.17.0",
+    "integration": "Directory",
+    "check": "system.disk.directory.exists",
+    "statuses": ["ok", "warning"],
+    "groups": [],
```

Comment:
```suggestion
    "groups": ["host"],
```

Reply:
Changed, how do `groups` should be populated? Is it `host` because this is a service check on a system integration, so host related?